tests/experimental/plotting/wrappers_test.py

```diff
@@ -7,37 +7,37 @@
 
 
 def test_plot_ndarray():
-    sc.plot(np.arange(50.))
+    sc.plot(np.arange(50.))._to_widget()
 
 
 def test_plot_variable():
-    sc.plot(sc.arange('x', 50.))
+    sc.plot(sc.arange('x', 50.))._to_widget()
 
 
 def test_plot_data_array():
-    sc.plot(make_dense_data_array(ndim=1))
+    sc.plot(make_dense_data_array(ndim=1))._to_widget()
     da = make_dense_data_array(ndim=2)
-    sc.plot(da)
-    da.plot()
+    sc.plot(da)._to_widget()
+    da.plot()._to_widget()
 
 
 def test_plot_data_array_missing_coords():
-    sc.data.table_xyz(100).plot()
+    sc.data.table_xyz(100).plot()._to_widget()
 
 
 def test_plot_dataset():
     ds = make_dense_dataset(ndim=1)
-    sc.plot(ds)
+    sc.plot(ds)._to_widget()
 
 
 def test_plot_dict_of_ndarrays():
-    sc.plot({'a': np.arange(50.), 'b': np.arange(60.)})
+    sc.plot({'a': np.arange(50.), 'b': np.arange(60.)})._to_widget()
 
 
 def test_plot_dict_of_variables():
-    sc.plot({'a': sc.arange('x', 50.), 'b': sc.arange('x', 60.)})
+    sc.plot({'a': sc.arange('x', 50.), 'b': sc.arange('x', 60.)})._to_widget()
 
 
 def test_plot_dict_of_data_arrays():
     ds = make_dense_dataset(ndim=1)
-    sc.plot({'a': ds['a'], 'b': ds['b']})
+    sc.plot({'a': ds['a'], 'b': ds['b']})._to_widget()
```

Comment:
Is there value in making `_to_widget` public? That is, can it help users to compose their own widgets?

Reply:
yeah, maybe we should just do that :+1: